osd/OCPBUGS8038_4_12_install_failure.json

```diff
@@ -0,0 +1,7 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "summary": "Installation blocked, action required",
+    "description": "Your cluster's installation is blocked because of an installation bug described in https://issues.redhat.com/browse/OCPBUGS-8038. There is currently no way to reliably ensure an installation will succeed until the bug is resolved. Red Hat is working on resolving the issue. Please retry your cluster installation.",
```

Comment:
maybe we could parametrized the bug id part and then add as well a short explanation of the bug in the text to avoid them to have to look into the jira

Reply:
Just to clarify: I modified the template to have a parametrized link to the OCPBUGS ticket.